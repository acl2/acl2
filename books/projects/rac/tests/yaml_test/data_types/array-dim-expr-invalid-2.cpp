// RAC begin

int foo(int n) {
  int a[1 + n] = {0, 0, 0};
  return 0;
}
