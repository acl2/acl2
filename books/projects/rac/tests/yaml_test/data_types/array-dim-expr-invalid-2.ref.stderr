array-dim-expr-invalid-2.cpp:4 (8-13): Invalid array size (it shoud be a constant, stricly positive expression)
 4 |   int a[1 + n] = {0, 0, 0};
   |         ^^^^^
